extensions/Webhooks/template/en/default/account/prefs/webhooks.html.tmpl

```diff
@@ -156,15 +146,23 @@ window.onload = function() {
       <td>[% webhook.product.name FILTER html %]</td>
       <td>[% webhook.component ? webhook.component.name : 'Any' FILTER html %]</td>
       <td>
-        [% connector = connectors.by_name('Webhook_' _ webhook.id)
-           config = connector.config
-        %]
+        [% config = connector.config %]
         <select name="[% connector.name FILTER html %].enabled"
                 id="[% connector.name FILTER html %]_enabled">
           <option value="Enabled" [% 'selected' IF config.${'enabled'} == 'Enabled' %]>Enabled</option>
           <option value="Disabled" [% 'selected' IF config.${'enabled'} == 'Disabled' %]>Disabled</option>
         </select>
       </td>
+      <td>
+        [% IF connector.backlog.count%]
+          <a href="page.cgi?id=webhooks_queues.html&amp;[% ~%]
+                  webhook=[% webhook.id FILTER uri %]">
+            [% connector.backlog.count FILTER html %]
+          </a>
+        [% ELSE %]
+          [% connector.backlog.count FILTER html %]
+        [% END %]
```

Comment:
If we do rename to **Errors** as mentioned above, I would just not display anything in the column row instead of 0.